css/main.css

```diff
@@ -1,11 +1,35 @@
+:root {
```

Comment:
creating a separate class called `lightTheme` would be better here than directly doing in :root . Like you did for `darkTheme`.

Reply:
We can do that but when website initially loads (i.e. when it is not having a cookie) there will be no theme which will be applied to pages and it will be all black and white and if we do this by adding a class (like add a class when the site loads), so if someone has slow internet speed (we know that javascript loads in the end), so there will be delay in apply the theme for few miliiseconds which will lead to bad UX.